boring/src/ssl/mod.rs

```diff
@@ -1281,6 +1304,21 @@ impl SslContextBuilder {
         }
     }
 
+    /// Gets the list of supported ciphers for protocols before TLSv1.3.
+    ///
+    /// See [`ciphers`] for details on the format
+    ///
+    /// This corresponds to [`SSL_CTX_get_ciphers`].
+    ///
+    /// [`ciphers`]: https://www.openssl.org/docs/manmaster/man1/ciphers.html
+    /// [`SSL_CTX_set_cipher_list`]: https://www.openssl.org/docs/manmaster/man3/SSL_CTX_get_ciphers.html
+    pub fn get_ciphers(&mut self) -> Result<&StackRef<SslCipher>, ErrorStack> {
```

Comment:
This should take `&self` and you may want to put the same method on `SslContext` too.